Extension/src/LanguageServer/settings.ts

```diff
@@ -461,6 +463,33 @@ export class CppSettings extends Settings {
         return super.Section.get<boolean>("inlayHints.referenceOperator.showSpace") === true;
     }
 
+    public get otfDocsEnabled(): PromiseLike<boolean> {
+        // Check if the user has access to copilot.
+        return vscode.lm.selectChatModels({vendor: "copilot"}).then((models) => {
```

Comment:
This is a slightly different language model check than the one used in `onOTFDocs()`. I'd extract a shared helper function to select the language model to ensure we don't end up in a scenario where this check passes and shows the link, but then clicking the link is guaranteed to fail.